src/components/SearchFacetsControl/index.tsx

```diff
@@ -72,11 +69,11 @@ const SearchFacetsControl = () => {
           }
 
           if (facet) {
-            const isPresent = !!searchFacets.find(v => v.name === facet.name)
+            const disabled = !facet.operatorTypes && !!searchFacets.find(v => v.name === facet.name)
```

Comment:
I tried to generalize the facets that should only be used once. Let me know if I should expand this logic.